CMakeLists.txt

```diff
@@ -85,6 +85,8 @@ if((${CMAKE_PROJECT_NAME} STREQUAL ${PROJECT_NAME} OR ${PROJECT_NAME}_BUILD_TEST
     set(ocpp_BUILD_TESTING ON)
     set(LIBEVSE_SECURITY_BUILD_TESTING ON)
     set(everest-evse_security_BUILD_TESTING ON)
+    set(EVEREST_SQLITE_BUILD_TESTING ON)
```

Comment:
shouldnt be required because it should be set internally by `everest-sqlite_BUILD_TESTING` 
```suggestion
```